src/io/calimero/dptxlator/DPTXlator.java

```diff
@@ -506,7 +506,7 @@ final String removeUnit(final String value)
 		return value.trim();
 	}
 
-	static final short ubyte(final int value)
+	static short ubyte(final int value)
```

Comment:
Putting final on static methods prevents hiding the method, its usage is fine.

Reply:
Rolled back as requested. AFAIR there are methods only declared static without the final keyword. Not sure if this is intended then.